src/ActionInitialization.cpp

```diff
@@ -21,6 +22,7 @@ ActionInitialization::~ActionInitialization()
 
 void ActionInitialization::Build() const
 {
+  // std::cout << "Source name: " << m_sourceName << std::endl;
```

Comment:
Please avoid merging commented-out debug statements

Reply:
It was overlooked. Removed now.